libass/ass.h

```diff
@@ -223,7 +223,7 @@ typedef enum {
      * purposes that do not involve actual ASS subtitles authored for
      * distribution.
      */
-    ASS_FEATURE_INCOMPATIBLE_EXTENSIONS,
+    ASS_FEATURE_INCOMPATIBLE_EXTENSIONS = 1,
```

Comment:
You must not change public enum values, this breaks ABI.
So `0` and `1 = 1 << 0`, which fortunately works well with a bit mask.

Reply:
Yes, and to expand a bit, _this_ (public) enum is not expected to be bit-flags. It’s just an enum. The bit stuff should remain an encapsulated implementation detail.

Reply:
So we can either use `1 << feature` in the bit-mask ([example](https://gist.github.com/MrSmile/ef9c0118163e46a0ba8e7a5529d6ead1/bcfca1a4e724acb4aa1911ff8406bcedc0b59630), but it needs an overflow check for the shift) or create a separate (internal) enum with bit-flags ([example](https://gist.github.com/MrSmile/ef9c0118163e46a0ba8e7a5529d6ead1/0ab3bb1ff8af4abdaef52d79f87c9989f4d35411)).